### PR TITLE
Save screenshots to app private storage

### PR DESCRIPTION
**Pull request by attilakiss323**

### CODE REVIEW

**Review by majeed-platogo — CHANGES_REQUESTED** [review]
Since permission is no longer need on android you need to remove [this](https://github.com/platogo/cordova-screenshot/blob/f65e641cd2178d7a56d36051065571dc6712a6f2/src/android/Screenshot.java#L73)

File: src/android/ScreenshotSaver.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package com.darktalker.cordova.screenshot;
 
+import static com.xtreme.plugins.XtremePushPlugin.TAG;
```
Comment (dmitry-platogo) [inline src/android/ScreenshotSaver.java]: Could you please clarify, why do we need this line? and how does XtremePushPlugin connected to it?

Shouldn't we just define Tag here as smth like:
```
public class ScreenshotSaver {
  private static final String TAG = "ScreenshotSaver";
```

**Review by dmitry-platogo — APPROVED** [review]
I was able to see screenshots in the app after this fix, great job @attilakiss323

File: src/android/Screenshot.java [hunk 2]
```diff
@@ -70,11 +70,7 @@ public boolean execute(String action, JSONArray args, final CallbackContext call
             if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
```
Comment (majeed-platogo) [inline src/android/Screenshot.java]: No need for the conditional statement here I think, just call `saveScreenshot` @attilakiss323

File: src/android/ScreenshotSaver.java [hunk 3]
```diff
@@ -52,25 +56,21 @@ public ScreenshotSaver(CordovaInterface cordova, View view, String fileName, Cal
...
+                    FileOutputStream fos = new FileOutputStream(storageDir);
                     bitmap.compress(Bitmap.CompressFormat.JPEG, 100, fos);
-
                     JSONObject jsonRes = new JSONObject();
-                    jsonRes.put("filePath", imageUri);
+                    jsonRes.put("filePath", storageDir.getAbsolutePath());
                     PluginResult result = new PluginResult(PluginResult.Status.OK, jsonRes);
                     pluginContext.sendPluginResult(result);
                     fos.close();
+                    Log.d(TAG, "Screenshot saved as: " + storageDir.getAbsolutePath());
```
Comment (majeed-platogo) [inline src/android/ScreenshotSaver.java]: I think we should remove the `Log.d` before merging

**Review by majeed-platogo — CHANGES_REQUESTED** [review]
@attilakiss323 @varsha-platogo everything currently looks good, great job. I've added a couple of comments that I think needs addressing

File: src/android/ScreenshotSaver.java [hunk 4]
```diff
@@ -14,6 +14,7 @@
 import android.view.PixelCopy;
 import android.view.View;
 import android.util.Base64;
+import android.util.Log;
```
Comment (majeed-platogo) [inline src/android/ScreenshotSaver.java]: @attilakiss323 small thing here, please remove unused import

**Review by majeed-platogo — CHANGES_REQUESTED** [review]

**Review by majeed-platogo — APPROVED** [review]
Nice work @attilakiss323 @varsha-platogo

**Pull request merged**